pkg/config/config.go

```diff
@@ -56,7 +56,7 @@ func init() {
 }
 
 func (operatorConfig *NSXOperatorConfig) HAEnabled() bool {
-	if operatorConfig.EnableHA == nil || *operatorConfig.EnableHA == true {
```

Comment:
operatorConfig.EnableHA == nil, this one is ok, 
operatorConfig.EnableHA is from ncp configmap, HA is default enabled even if there is no user setting. 
So, operatorConfig.EnableHA == nil, which means using default value: enabled.